optional/clamav/Dockerfile

```diff
@@ -4,8 +4,11 @@ RUN apk add --no-cache clamav rsyslog wget clamav-libunrar
 
 COPY conf /etc/clamav
 COPY start.sh /start.sh
+COPY health.sh /health.sh
 
 EXPOSE 3310/tcp
 VOLUME ["/data"]
 
 CMD ["/start.sh"]
+
+HEALTHCHECK CMD /health.sh
```

Comment:
Couldn't we just use `rc-status clamav` to check it's health? It returns autmatically a nen-zero exit code when somethings wrong